arangod/Cluster/ClusterInfo.cpp

```diff
@@ -1950,23 +1987,22 @@ int ClusterInfo::dropCollectionCoordinator(std::string const& dbName,
   if (numberOfShards == 0) {
     loadCurrent();
     events::DropCollection(collectionID, TRI_ERROR_NO_ERROR);
-    return TRI_ERROR_NO_ERROR;
+
+    return Result(TRI_ERROR_NO_ERROR);
   }
 
   {
     CONDITION_LOCKER(locker, agencyCallback->_cv);
 
     while (true) {
-      errorMsg = *errMsg;
-
       if (*dbServerResult >= 0) {
         cbGuard.fire();  // unregister cb before calling ac.removeValues(...)
         // ...remove the entire directory for the collection
         ac.removeValues("Current/Collections/" + dbName + "/" + collectionID, true);
         loadCurrent();
         events::DropCollection(collectionID, *dbServerResult);
 
-        return *dbServerResult;
+        return Result(*dbServerResult, *errMsg);
```

Comment:
is `errMsg` still populated here or can it be removed?

Reply:
this variable does not seem to be used